service/parserprovider/properties.go

```diff
@@ -26,28 +25,31 @@ import (
 	"go.opentelemetry.io/collector/config"
 )
 
-type setFlagMapProvider struct{}
+type propertiesMapProvider struct {
+	properties []string
+}
 
-// NewPropertiesMapProvider returns a MapProvider, that provides a config.Map from set flag(s) properties.
+// NewPropertiesMapProvider returns a MapProvider, that provides a config.Map from the given properties.
 //
-// The implementation reads --set flag(s) from the cmd and concatenates them as a "properties" file.
-// Then the properties file is read and properties are set to the loaded Parser.
-func NewPropertiesMapProvider() MapProvider {
-	return &setFlagMapProvider{}
+// Properties must follow the Java properties format, key-value list separated by equal sign with a "."
```

Comment:
Why follow java properties format here? I'm not against it just curious

Reply:
This was already the format, I just clarified.